src/endpoint/endpoint.cpp

```diff
@@ -1,17 +1,13 @@
-//
-// Created by zanna on 05/10/18.
-//
-
 #include "endpoint.h"
 
 classifier::Classifier endpoint::Endpoint::classifier_;
 // TODO retrieve roulette address
-db::DBQuery* endpoint::Endpoint::roulette_ = new db::DBQuery("localhost", 57684);
-
+db::DBQuery *endpoint::Endpoint::roulette_ = new db::DBQuery("localhost",
+                                                             57684);
```

Comment:
We should put this variables as part of the CMake configuration

Reply:
Do you mean the pointer or localhost/57684?

Reply:
No, I mean something like
```C++
#define ROULETTE_DEFAULT_ADDRESS localhost
#define ROULETTE_DEFAULT_PORT 57687
```

In the `config.h.in` file